contracts/authorizers/RequesterAuthorizer.sol

```diff
@@ -234,114 +217,16 @@ abstract contract RequesterAuthorizer is ERC2771Context, IRequesterAuthorizer {
 
     /// @notice Verifies the authorization status of a request
     /// @param airnode Airnode address
-    /// @param endpointId Endpoint ID
     /// @param requester Requester address
     /// @return Authorization status of the request
     function isAuthorized(
         address airnode,
-        bytes32 endpointId,
         address requester
     ) public view override returns (bool) {
-        return
-            userIsAuthorized(deriveServiceId(airnode, bytes32(0)), requester) ||
-            userIsAuthorized(deriveServiceId(airnode, endpointId), requester);
-    }
-
-    /// @notice Verifies the authorization status of a request
-    /// @dev This method has redundant arguments because V0 authorizer
-    /// contracts have to have the same interface and potential authorizer
-    /// contracts may require to access the arguments that are redundant here
-    /// @param requestId Request ID
-    /// @param airnode Airnode address
-    /// @param endpointId Endpoint ID
-    /// @param sponsor Sponsor address
-    /// @param requester Requester address
-    /// @return Authorization status of the request
-    function isAuthorizedV0(
```

Comment:
We now have authorizer types specified in [`config.chains.authorizers`](https://docs.api3.org/airnode/v0.9/reference/deployment-files/config-json.html#authorizers) so even if we wanted to use this with a V0 Airnode we wouldn't need it to satisfy this interface